R/WINDOWS.md

```diff
@@ -20,25 +20,28 @@ license: |
 
 To build SparkR on Windows, the following steps are required
 
-1. Install R (>= 3.1) and [Rtools](https://cloud.r-project.org/bin/windows/Rtools/). Make sure to
+1. Make sure `bash` is available and in `PATH` if you already have a built-in `bash` on Windows. If you do not have, install [Cygwin](https://www.cygwin.com/).
```

Comment:
btw, do we know Cygwin still work? or should we tell ppl to use https://docs.microsoft.com/en-us/windows/wsl/install-win10 ?

Reply:
At least it works on my laptop - I have one Windows laptop that has Spark dev setup. Not sure if it still works on latest Windows though. Seem still being actively developed.